.azure-pipelines.yml

```diff
@@ -487,8 +487,11 @@ jobs:
     parallel: 10 # chosen to make runtime ~2h
   timeoutInMinutes: 180
   pool:
-    name: 'Hosted Windows Client'
+    vmImage: 'windows-2019'
```

Comment:
@foolip - can you educate me a bit on the 'Hosted Windows Client' pool? I initially tried to use it, but found that it didn't have py3 available and I wasn't sure how to make it available (UsePythonVersion isn't a task there, and `tools/ci/azure/install_python.yml` seems to *manually* download py2.7 and install it? :O). I tried to poke around azure pipelines, but I don't have access to the web-platform-tests organization - I only have access to the wpt pipeline itself.

Reply:
This pool is maintained by @mustjab and updated weekly by a process that's opaque to us (outside Microsoft) but has worked pretty reliably so far. To get additional software installed by default, @mustjab would have to help.

If we find that windows-2019 works to run the tests now, we should actually switch to that. When we set this up, none of the agents available by default could launch Edge classic. Perhaps now it's possible to run the new Edge in one of these VMs.

I'll try to add you to the web-platform-tests org in Azure DevOps.

Reply:
You should be able to update install_python.yml to add a task to download and install py3. Do we want to install specific py3 version? Do you have a link to py3 MSI installer?

Reply:
@mustjab - I wasn't able to locate an amd64 msi at https://www.python.org/ftp/python/3.8.5/ . There's a subdirectory, https://www.python.org/ftp/python/3.8.5/amd64/, with **many** msi files - but not sure which (if any) are correct! :/

Reply:
Looks like they ship them as .exe files now, so a link like this would work: https://www.python.org/ftp/python/3.7.9/python-3.7.9-amd64.exe and you can use this guide on how to run this installer without UI: https://docs.python.org/3.7/using/windows.html